packages/nextjs/utils/Constants.ts

```diff
@@ -0,0 +1,490 @@
+// Eth
+const universalEthAddress =
+  "0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7" as const;
+
+const devnetEthClassHash =
+  "0x046ded64ae2dead6448e247234bab192a9c483644395b66f2155f2614e5804b0" as const;
+
+const sepoliaMainnetEthClassHash =
+  "0x07f3777c99f3700505ea966676aac4a0d692c2a9f5e667f4c606b51ca1dd3420" as const;
+
+// Strk
+const universalStrkAddress =
+  "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d" as const;
+
+const sepoliaMainnetStrkClassHash =
+  "0x04ad3c1dc8413453db314497945b6903e1c766495a1e60492d44da9c2a986e4b" as const;
+
+const devnetStrkClassHash =
+  "0x046ded64ae2dead6448e247234bab192a9c483644395b66f2155f2614e5804b0" as const;
+
+const devnetAbi = [
```

Comment:
lets remove the `devnetAbi`, and use the `sepoliaAbi` as universal one and call it `universalErc20Abi` @oceanlvr @Nadai2010

Reply:
sir removed devnetAbi and use universalErc20Abi